hedera-node/hedera-app/src/test/java/com/hedera/node/app/state/merkle/MerkleSchemaRegistryTest.java

```diff
@@ -63,6 +64,8 @@ void setUp() {
         config = Mockito.mock(Configuration.class);
         final var hederaConfig = Mockito.mock(HederaConfig.class);
         lenient().when(config.getConfigData(HederaConfig.class)).thenReturn(hederaConfig);
+        // Use a fresh MerkleDb instance for every test run
+        MerkleDb.setDefaultPath(null);
```

Comment:
Could the default path be set to a temporary path (perhaps using JUnit `@TempDir` annotation) so that test data is automatically cleaned up?

Reply:
I think it would work with `@TempDir`, let me try...